### PR TITLE
Functional tests - Fix nightly 8.1.x : Fix slider URL

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Fix nightly 8.1.x : Fix slider URL
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Ci and nightly are :green_apple: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/7568789864
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/classic/06_homePage/01_checkSlider.ts [hunk 1]
```diff
@@ -61,8 +61,6 @@ describe('FO - Home Page : Check slider', async () => {
     await testContext.addContextItem(this, 'testIdentifier', 'checkSliderURL', baseContext);
 
     const currentURL = await homePage.getSliderURL(page);
-    await expect(currentURL)
-      .to.contains('www.prestashop-project.org')
-      .and.to.contains('homeslider&utm_campaign=back-office-EN&utm_content=download');
+    await expect(currentURL).to.contains('www.prestashop-project.org');
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/classic/06_homePage/01_checkSlider.ts:64]: @nesrineabdmouleh Why this change ? Isn't it a regression ?

**Review by boubkerbribri — APPROVED** [review]

File: tests/UI/campaigns/functional/FO/classic/06_homePage/01_checkSlider.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/FO/classic/06_homePage/01_checkSlider.ts:64]: I don't know why it's changed but I think it's not a problem it's just a link for the slide

**Review by jolelievre — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]
